.github/CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 
 ### Breaking changes
 
+* Update MacOS wheel builds to require Monterey (12.0) or greater for x86_64 and ARM.
```

Comment:
To keep the consistency throughout the changelog:
```suggestion
* Update MacOS wheel builds to require Monterey (12.0) or greater for x86_64 and AArch64.
```

Reply:
MacOS uses ARM as the arch name, not aarch64 ;)

Reply:
Hmm.. right! then we need to update it to ARM64 here and in other places :smile:

```suggestion
* Update MacOS wheel builds to require Monterey (12.0) or greater for x86_64 and ARM64.
```

Let's get this PR in, we can update it when reviewing the CHANGELOG before the next release